Haskell/MaîtreD.hs

```diff
@@ -1,11 +1,19 @@
 module MaîtreD where
 
-import Data.Time (ZonedTime)
+import Data.Time (ZonedTime (zonedTimeToLocalTime, zonedTimeZone))
```

Comment:
Either is fine, but I think I've only seen the `ZonedTime (..)` syntax before, so it genuinely took me a while to figure out the reason...